src/PKSim.Core/Model/IndividualFactory.cs

```diff
@@ -73,8 +73,8 @@ private Individual createStandardIndividual(OriginData originData, Func<IIndivid
          var individual = _objectBaseFactory.Create<Individual>();
          individual.OriginData = originData;
 
-         //default icon for an individual is the name of the species
-         individual.Icon = originData.Species.Name;
+         //default icon for an individual is the icon of the species
+         individual.Icon = originData.Species.Icon;
```

Comment:
@Yuri05 I am not sure why this was done like that. Assuming that we did not have an icon property at the beginning